File: local-data-persistent/LatihanSQLite/java/app/src/main/java/com/dicoding/picodiploma/mynotesapp/db/DatabaseContract.java
package com.dicoding.picodiploma.mynotesapp.db;

import android.provider.BaseColumns;

/**
 * Created by dicoding on 10/12/2017.
 */

public class DatabaseContract {

    public static final class NoteColumns implements BaseColumns {
        public static final String TABLE_NAME = "note";

        //Note title
        public static final String TITLE = "title";
        //Note description
        public static final String DESCRIPTION = "description";
        //Note date
        public static final String DATE = "date";

    }
}

File: local-data-persistent/LatihanSQLite/kotlin/app/src/main/java/com/dicoding/picodiploma/mynotesapp/db/DatabaseContract.kt
package com.dicoding.picodiploma.mynotesapp.db

import android.provider.BaseColumns

/**
 * Created by dicoding on 10/12/2017.
 */

internal class DatabaseContract {

    internal class NoteColumns : BaseColumns {
        companion object {
            const val TABLE_NAME = "note"
            const val _ID = "_id"
            const val TITLE = "title"
            const val DESCRIPTION = "description"
            const val DATE = "date"
        }

    }
}
